src/ts/components/core/tooltip/FloatingTooltip.tsx

```diff
@@ -15,7 +15,7 @@ interface Props extends TooltipBaseProps, DashBaseProps {
 const FloatingTooltip = (props: Props) => {
     const { children, boxWrapperProps, setProps, loading_state, ...others } =
         props;
-    const boxProps = { w: "fit-content", ...boxWrapperProps };
+    const boxProps = { w: "fit-content", key: "tooltip-target", ...boxWrapperProps };
```

Comment:
Is there ever a case you could get two of these boxes as siblings? It's unclear to me why this is necessary at all given the nesting structure of the react components it generates, `Box` is the only child of `Tooltip.Floating`, so why would it need a key? This makes me speculate there may be something funny going on when Mantine renders this... and two siblings with the same key could be worse than no key at all. Two components with tooltips next to each other? Two tooltips on the same component?

Reply:
Yes, I'm concerned about this as well.  Do you have any other suggestions?  I did try an app with a few tooltips on a page and it worked fine.  I'll try it with two tooltips on the same component.

Reply:
Actually, you can't have 2 tooltips on the same component because the Tootlip `children` must be a single component:

```
dmc.Tooltip(
            id="my-tooltip",
            children=dmc.Button("some text", id="my-text"),
            label="Tooltip text",
        ),
```

Reply:
OK - let's leave it then. In principle we could do a random number via `useState` or some such so that it's retained between renders... but it doesn't seem like this complexity is necessary.